maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/HardshipOrchestrationServiceTest.java

```diff
@@ -371,6 +374,8 @@ void givenCrownCourt_whenUpdateIsInvoked_thenApplicationDTOIsUpdatedWithNewHards
         applicationDTO.getCrownCourtOverviewDTO().setContribution(contributionsDTO);
         applicationDTO.getAssessmentDTO().getFinancialAssessmentDTO().getHardship().getCrownCourtHardship()
                 .setSolictorsCosts(TestModelDataBuilder.getHRSolicitorsCostsDTO());
+        RepOrderDTO repOrderDTO = TestModelDataBuilder.getTestRepOrderDTO(applicationDTO);
```

Comment:
Can we could consolidate some of the repeated stub definitions into a `@BeforeAll` or other dedicated method to cut down on some of the duplication please.

Reply:
Done